flow-typed/npm/tape-cup_v4.x.x.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable  */
```

Comment:
As of right now, the plan is to include this libdef file in all of our plugins.  Ideally in the future, it will either be supplied by:

* [flow-typed](https://github.com/flowtype/flow-typed) which already includes type definitions for `tape` (which is what this is based upon)
* `tape-cup` package itself, and then included similar to [how `fusion-core` is today](https://github.com/fusionjs/fusion-tokens/blob/master/.flowconfig#L7).

If we are the only consumers of `tape-cup`, the second option provides us with the most flexibility as we will own the entire pipeline.  It also makes it easier to combine `tape-cup` releases with type definition updates.  Will have to follow up with @rtsao when he is back.

Reply:
I feel like we should just have `tape-cup` export a `flow.js` file. I feel like we will be adding a bunch of tech debt by copying this everywhere...

Reply:
Agreed, and that is the 2nd option above.  For now, I say we do this and export from `tape-cup` once Ryan is back.  Alternatives are outlined in the [Shipping type definitions](https://docs.google.com/document/d/12tLKnrCeK9hBE3guKfeYGRZeQUvQn-E6OgSjaNHnrhs/edit#heading=h.xfjinvc98ip9) section of the RFC.